validator/client/runner.go

```diff
@@ -115,6 +116,7 @@ func run(ctx context.Context, v Validator) {
 				wg.Wait()
 				span.End()
 			}()
+			v.LogAttestationsSubmitted()
```

Comment:
Shouldn't this be after the `span.End()` above?

That said, looking at the code I don't see where the goroutines call `wg.Done()` so not sure that goroutine ever gets past `wg.Wait()`.

Reply:
It doesn't get logged if it's:
```go
go func() {
	wg.Wait()
	v.LogAttestationsSubmitted()
	span.End()
}()
```

Any suggestions?

Reply:
Try adding `wg.Done()` at line 111 and see if that helps.

Reply:
Thanks! 👍